src/templates/sidebar.html

```diff
@@ -1,302 +1,525 @@
-            <img src="/assets/images/helpcenter.png">
+        </li>
+
+        <li>
+          <a href="/developer/">
+            Dev Center Home
+            <br /><small>Build on our SDKs, APIs and JS</small>
           </a>
-        </div>
-        <div class="docs-searchbox-full">
-            <form class="js-search-form form-search" action="/search/">
-              <div class="input-append">
-                <input type="text" class="search-query rounded" placeholder="Search" name="q">
-              </div>
-            </form>
-        </div>
-        <li><a href="/guides/?isSuccessCenter=true"><i class="fa fa-fw fa-life-ring"></i> SaaSquatch Guides</a></li>
-        <li class="Divider">Administration Portal</li>
-                <li><a href="/success/using-referral-saasquatch">Using the SaaSquatch Portal</a></li>
-                <li><a href="/success/navigating-the-portal/">Navigating the SaaSquatch Portal</a></li>
-                <li><a href="/guides/one-time">Manual User Actions: Add a Reward, Referral or Event</a></li> 
-                <li><a href="/designer/email-editor">Design Program Emails</a></li>
-                <li><a href="/designer/widget-editor">Customize Program Widgets</a></li>
-        <li class="Divider">Program Reference</li>
-                <li><a href="/growth/quickstart">Growth Automation Program General Quickstart</a></li>
-                <li><a href="/growth/ga-mechanisms">Growth Automation Program Mechanisms</a></li>
-                <li><a href="/guides/referral-quickstart">Growth Automation Referral Program - Quickstart</a></li>
-                <li><a href="/developer/purchase-object">Growth Automation Purchase Event</a></li>
-        <li class="Divider">Program Resources</li>
-                <li><a href="/features/user-segmentation">User Segmentation</a></li>
-                <li><a href="/features/program-i18n">Program Internationalization</a></li>
-                <li><a href="/success/gift-card-rewards">Gift Card Rewards</a></li>
-                <li><a href="/features/rewards-fuel-tank">Custom Rewards Fuel Tank</a></li>
-                <li><a href="/designer/short-tags">Email Template Short Tags</a></li>
-        
-        <li><a href="/integrations/"><i class="fa fa-fw fa-cubes"></i> Integrations</a></li>
-    </ul>
-</li>
-            </ul>
         </li>
-        <li class="developerCenter">
-            <img class="sqtch-icon" src="/assets/images/Saasquatch-icons-cog.svg">
-            <span>
-              Developer Center <br>
-            </span>
-            <ul>
+        <li>
+          <a href="/guides/"
+            ><i class="fa fa-fw small fa-code"></i> Dev Guides</a
+          >
+        </li>
+        <li>
+          <a href="/topics/json-web-tokens"
+            ><i class="fa small fa-fw fa-lock"></i> JSON Web Tokens</a
+          >
+        </li>
+        <li>
+          <a href="/developer/testing"
+            ><i class="fa fa-fw small fa-file"></i> Testing Best Practices</a
```

Comment:
More iconic for testing.

```suggestion
            ><i class="fa fa-fw small fa-flask"></i> Testing Best Practices</a
```